prime-router/docker-compose.yml

```diff
@@ -32,6 +32,7 @@ services:
       - OKTA_baseUrl=hhs-prime.okta.com
       - OKTA_clientId=0oa6fm8j4G1xfrthd4h6
       - OKTA_redirect=http://localhost:7071/api/download
+      # - TokenSigningSecret=<generate secret using ReportStreamSecretFinder.main>
```

Comment:
A few questions:
* Is this something we intend to keep static for all developers or generate like we do the Hashicorp Vault secrets?
* Could we use the `.env` to populate, so each developer doesn't have to modify their `docker-compose.yml`?
* Is this something that could be stored in the Hashicorp Vault / Azure Key Vault?

Reply:
I see later in the code we're loading from the vault? Why an envvar then?

Reply:
As an initial improvement,I have moved this out of the docker-compose file and am creating it in the vault init.sh script, which is nicer, as the user doesn't have to think about it, but needs further improvement.

This secret is being loaded using the secrets store, which is implemented using env vars locally (EnvVarSecretService.kt), hence the env var.   In Azure, it will be in the Azure secrets store.    We could use the credentials store, but its not really a credential, since there's no username, so the right way to do this would be to create a new credential type.  That's a todo at the moment.